### DISCUSSION TITLE
I have uninstalled the OCR workflow, but keep getting Notification errors in the Nextloud Log at INFO level.

### DISCUSSION THREAD

**Discussion by richard378** [R0Wi-DEV/workflow_ocr#252] [Q&A]
Hi,
How do I remove notifications for OCR workflow when OCR workflow is uninstalled.  Why am I getting notification errors in workflowengine for the ocr app when it is not installed?

Nextlcloud 29.01

`{"reqId":"vFr9Y5FhLtg2NXDkzhSr","level":1,"time":"2024-06-01T07:33:32+00:00","remoteAddr":"xxx.xxx.x.xxx","user":"xxxx@xxxxx.com","app":"no app in context","method":"GET","url":"/ocs/v2.php/apps/notifications/api/v2/notifications?format=json","message":"Notification was not parsed by any notifier [app: workflow_ocr, subject: ocr_error]","userAgent":"Mozilla/5.0 (Windows) mirall/3.13.0stable-Win64 (build 20240423) (Nextcloud, windows-10.0.22631 ClientArchitecture: x86_64 OsArchitecture: x86_64)","version":"29.0.1.1","data":[],"id":"665aced1acc8f"}`

**Comment by R0Wi** [comment]
Hey, first of all it's a pity that you uninstalled this app :cry: 

Nevertheless you should be able to get rid of all unprocessed notifications created by the `workflow_ocr` app, by using the following SQL statement:

```sql
DELETE
FROM oc_notifications
WHERE app = 'workflow_ocr'
```
I guess it would be a good idea if we add this to the uninstall routine

**Reply by richard378** [reply]
I uninstalled it only temporarily.  I have been testing it and I have since reinstalled it.  But wondered at the error.  Thank you for the information on the fix.
